### DISCUSSION TITLE
Add an option to remove the default language from the front-end url

### DISCUSSION THREAD

**Discussion by jf-viguier** [PrestaShop/PrestaShop#31482] [Ideas]
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.

### Is your feature request related to a problem?

When a shop with only one language add a second language, all old urls are redirected to /lang/page.
By example /mypage is redirected to /en/mypage.
Google doesn't like redirections so much.

### Describe the solution you'd like

Add an option to remove the default language from the front-end url.
Shop with default language en, mypage url is /mypage, extra language url is /fr/mypage and so on.

### Do you plan to work on this feature?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [ ] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by hibatallahAouadni** [comment]
Hello @jf-viguier 


Thank you for your suggestion. The Product Team will take it into consideration for future developments. 
Please be aware that there is no guarantee that this feature will be developed anytime soon.
So we're waiting for your PR 🚀 

Thank you

**Comment by jf-viguier** [comment]
Should the new option in SEO & URL menu @MatShir ?
